authors/apps/articles/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+from django.utils.text import slugify
+
```

Comment:
Consider eliminating some of these blank lines

Reply:
done